### ISSUE TITLE
[hackerone] safebrowsing setting not working

### ISSUE THREAD

**Issue by diracdeltas** [brave/browser-android-tabs#475]
Reported on https://hackerone.com/reports/294513:

```
com.brave.browser manifest does not enable safe browsing a feature implemented 22 June 2017 by google to strengthen WebView security.

POC: Go to www.testsafebrowsing.appspot.com and try any of the links with the Android version of the Brave browser. There will be no warning. Go to the links in Chrome Android browser and there's a warning. 

Fix:

Add the following line to your manifest.

<meta-data android:name="android.webkit.WebView.EnableSafeBrowsing"
android:value="true" />

In conclusion this is a great feature to enable to keep all users safer. Please let me know if you have any questions.
```

**Comment by diracdeltas** [comment]
talked to @SergeyZhukovsky  on slack.

1. the solution suggested by the hacker doesn't work because we are using native chromium webviews, not system webviews.
2. in the next release, the `Settings->Privacy->Safe Browsing` setting should be removed.

**Comment by AlexeyBarabash** [comment]
Option is disabled by PR https://github.com/brave/browser-android-tabs/pull/507

**Comment by LaurenWags** [comment]
Test Plan:
1. Clean install. Verify ```Settings->Privacy->Safe Browsing``` setting is removed.
2. Update from previous version.  Verify ```Settings->Privacy->Safe Browsing``` setting is removed.

cc @SergeyZhukovsky @AlexeyBarabash @diracdeltas for confirmation this is sufficient for testing

**Comment by AlexeyBarabash** [comment]
@LaurenWags , test plan looks good. Thanks you.
